### PR TITLE
Update presentations.md

### PR DESCRIPTION
**Pull request by packtpartner**
Added a new video listing

### CODE REVIEW

**Comment by judithpatudith** [comment]
@packtpartner it looks like this content is gated, and none of the other presentations on the list are. Do you have a non-gated version to link to instead?

**Comment by packtpartner** [comment]
No we do not have a non-gated version of this. But we Packt have the copyrights to this video.

**Comment by packtpartner** [comment]
Hi Judith,

Would it be possible to merger this PR?

**Comment by judithpatudith** [comment]
I'm checking with a committer now 🙂

**Comment by packtpartner** [comment]
Alright will wait for an update about it.

File: docs/presentations.md [hunk 1]
```diff
@@ -133,6 +133,10 @@ Presented Summer 2010 at [Yahoo](http://yahoo.com)
 # Apache Mesos - Training
 _(Listed in date descending order)_
 
+## Docker, Apache Mesos & DCOS: Run and manage cloud datacenter ([Video](https://www.packtpub.com/networking-and-servers/docker-apache-mesos-dcos-run-and-manage-cloud-datacenter-video))
+Manuj Aggarwal, TetraTutorials Team
```
Comment (greggomann) [inline docs/presentations.md]: Let's remove the "TetraTutorials Team" designation, for consistency.

Comment (packtpartner) [reply docs/presentations.md]: I have created another PR without TetraTutorials Team

**Comment by cf-natali** [comment]
Merged in https://github.com/apache/mesos/commit/f79a09a4e2caeaa215a3acc52b85ca50362ad171, closing.

**Pull request closed without merging**